app/src/test/java/ca/dal/cs/athletemonitor/athletemonitor/WorkoutDataTest.java

```diff
@@ -0,0 +1,45 @@
+package ca.dal.cs.athletemonitor.athletemonitor;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.Assert.assertTrue;
+
+/**
+ * Unit tests for the Team class
+ */
+
+public class WorkoutDataTest {
```

Comment:
Perhaps just delete this file since there's no tests in it. We'll add it again later when we see no coverage for Workout